content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
 
-## Standard API terminology
+Kubernetes relies on terminology from [ReST](https://en.wikipedia.org/wiki/Representational_state_transfer)
+to name and describe several API concepts:
 
-Most Kubernetes API resource types are [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects): they represent a concrete instance of a concept on the cluster, like a pod or namespace. A smaller number of API resource types are "virtual" - they often represent operations rather than objects, such as a permission check (use a POST with a JSON-encoded body of `SubjectAccessReview` to the `subjectaccessreviews` resource). All objects will have a unique name to allow idempotent creation and retrieval, but virtual resource types may not have unique names if they are not retrievable or do not rely on idempotency.
+* A **resource type** is the name used in the URL (for example: `pods`, `namespaces`, `services`);
+* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**;
+* A set of instances of a resource type is known as a **collection**;
+* A single instance of the resource type is called an **object**, or sometimes a **resource**.
 
-Kubernetes generally leverages standard RESTful terminology to describe the API concepts:
+{{< note >}}
+In URLs, Kubernetes types such as `pod` and `configmap` are **lowercase**. This is different from
+how Kubernetes conventionally renders those types elsewhere, such as the `.kind` field of an
+object, where the type is typically written in **UpperCamelCase**.
 
-* A **resource type** is the name used in the URL (`pods`, `namespaces`, `services`)
-* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**
-* A list of instances of a resource type is known as a **collection**
-* A single instance of the resource type is called a **resource**
+For example: `"kind": "ConfigMap"`.
+{{< /note >}}
 
-All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or to a namespace (`/apis/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource type will be deleted when its namespace is deleted and access to that resource type is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:
+All resource types are either scoped by the cluster (`/apis/<GROUP>/<VERSION>/*`) or to a namespace (`/apis/<GROUP>/<VERSION>/namespaces/<NAMESPACE>/*`). A namespace-scoped resource type will be deleted when its namespace is being deleted, and access to a namespaced resource type is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:
```

Comment:
Possible edit:
A namespace-scoped resource is deleted when its namespace is destroyed[removed], and access to a namespace-scoped resource is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:

Reply:
> A namespace-scoped resource type will be deleted when its namespace is being deleted

This statement makes no sense. Only resources are deleted, no the resource types.
@kbhawkey's suggestion gets the point.

Reply:
I've omitted this change from the latest revision.